### DISCUSSION TITLE
How do I merge two SVG files?

### DISCUSSION THREAD

**Discussion by martinfrandsen** [vvvv/SVG#959] [Q&A]
I got a svg template, which I wish to add a generated QR code to.
I can save the QR code as SVG file, and I can create a copy of the template, but I can't merge the two svg's together.

I thought it was as simple as taking the children from the QR code document and adding them to a SvgElement from the template, which I found using the "GetElementById" method.

**Comment by H1Gdev** [comment]
1. Get element with [GetElementById](https://github.com/svg-net/SVG/blob/a7e1e1226626d0dc6acd8f8854a9e1232580a020/Source/SvgDocument.cs#L216).
2. Get [Children](https://github.com/svg-net/SVG/blob/master/Source/SvgElement.cs#L218).
3. Add element with [Add](https://github.com/svg-net/SVG/blob/3daf279fc9357225eb151a263683c0885ebf2e63/Source/SvgElementCollection.cs#L87).
